lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -580,6 +582,7 @@ mixin _WheelEventListenerMixin on _BaseAdapter {
   }
 
   bool _isTrackpadEvent(DomWheelEvent event) {
+    if (browserEngine != BrowserEngine.firefox) return true;
```

Comment:
This was needed as my trackpad was not classified as trackpad as the `_isAcceleratedMouseWheelDelta` function returns true.

So for us to even add a heuristic for only trackpads, this function would have to be fixed first.